modules/claims_api/app/swagger/claims_api/v1/claims_controller_swagger.rb

```diff
@@ -185,16 +201,32 @@ class ClaimsControllerSwagger
             end
           end
 
-          response :default do
-            key :description, 'unexpected error'
+          response 401 do
+            key :description, 'Unauthorized'
+            content 'application/json' do
+              schema do
+                key :type, :object
+                key :required, [:errors]
+                property :errors do
+                  key :type, :array
+                  items do
+                    key :'$ref', :NotAuthorizedModel
+                  end
+                end
+              end
+            end
+          end
+
+          response 404 do
```

Comment:
What can cause this `claims` endpoint to return a 404? I would have assumed it was just an empty list if the Veteran doesn't have any established claims?

Reply:
I added that because of [this existing line](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/claims_api/app/controllers/claims_api/v1/claims_controller.rb#L21).

Reply:
weird... not what i would expect from a get list endpoint... fair enough